sys/suit/coap.c

```diff
@@ -340,6 +365,7 @@ static void _suit_handle_url(const char *url)
         riotboot_flashwrite_t writer;
 #ifdef MODULE_SUIT_V4
         suit_v4_manifest_t manifest;
+        manifest_ptr = &manifest;
```

Comment:
This would probably get an unused variable warning if not compiled with `MODULE_PROGRESS_BAR`

Reply:
I just checked (samr21, nrf52) and no error when progress_bar is not built.